### ISSUE TITLE
Input output mask does not correspond. 

### ISSUE THREAD

**Issue by lancesye** [nikhilroxtomar/FANet#1]
Shuffle set True in train_loader. In a new training epoch, the orders of  training sample will suffle,  not same as the last return_train_mask.

**Comment by nikhilroxtomar** [comment]
Thanks for valuable feedback, I have updated the code and set _shuffle=False_. The code would not work properly with the shuffle set as True.
